packages/excalidraw/tests/linearElementEditor.test.tsx

```diff
@@ -1099,7 +1099,7 @@ describe("Test Linear Elements", () => {
         .toMatchInlineSnapshot(`
           {
             "height": 130,
-            "width": "366.11716",
+            "width": "370.91788",
```

Comment:
The pointer to dimension calculation was simplified which led to dimensions difference in the case of linear elements. Which I believe shouldn't matter but I've updated the calculation to preserve previous behavior to avoid potential regressions.